expenses/expensesRouter.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+const expensesRouter = express.Router();
+
+const Expense = require('./expensesModel');
+
+expensesRouter.post('/', (request, response) => {
+  const expenseInfo = request.body;
+  const newExpense = new Expense(expenseInfo);
```

Comment:
You could also think about placing all this logic in your callback in its corresponding controller file.